site/_includes/icons/placeholder.svg

```diff
@@ -0,0 +1,37 @@
+{%- comment -%}
+  Usage: include icons/placeholder.svg args
+
+  args can be one of the following:
+    title: Used in the SVG `title` tag
+    text: The text to show in the image - default: 'width x height'
+    class: default: 'bd-placeholder-img'
+    color: The text color (foreground) - default: '#ddd'
+    background: The background color - default: '#777'
+    width: default: 100%
+    height: default: 180px
+    viewBox: default: not set
+{%- endcomment -%}
+
+{%- assign title = include.title | default: 'Generic Placeholder Image' -%}
+{%- assign class = include.class | default: '' -%}
+{%- assign color = include.color | default: '#ddd' -%}
+{%- assign background = include.background | default: '#777' -%}
```

Comment:
How about using colors from bootstrap grays color map?
https://getbootstrap.com/docs/4.1/getting-started/theming/#grays

Reply:
I'm mostly interested at getting this to work right first, which [isn't](https://github.com/twbs/bootstrap/pull/27633#issuecomment-439677194) atm.

Reply:
Can you help finding the issue? What is wrong with the `img` replacement? Sorry, I can't spot the issue 🕵🏻‍♂️
Is it the larger `font-size`? If yes, It's the extra `.bd-placeholder-img-lg` css class added to this placeholder.